README.md

```diff
@@ -39,72 +57,16 @@ A specification folder might look like the following:
   |     | - main.c
   |     | - Makefile
   | - test-04.03.sh     -> some script making use of the test-04.03 application
-  | - ....
+  | - ...
 /05-some-other-spec
   | - spec-05.md
   ...
 /README.md
 ```
 
-### Specs document
-
-The release specifications are descriptions of test-cases and certain other
-criteria that have to be tested/applied before each RIOT release. Each
-specification contains a description of it's goal as well as a number of tasks
-that have to be carried out (as much automated as possible).
-
-Each tasks is to be written in the style of a test-case. This means, that the
-following information must be given for each task:
-- pre-requisites (e.g. RIOT application used, hardware used, network topology, )
-- descriptions of how to exactly carry out the task (step-by-step if possible)
-- a precise list of passing/failing criteria, so that there is no doubt if a
-  task was successful or not
```

Comment:
Why remove this section?

Reply:
these three bullet points are down at the bottom, otherwise I trimmed information that I thought was obvious or unnecessary. Is there any removed information that doesn't meet that criteria that you can identify?